diff --git a/meson.build b/meson.build
--- a/meson.build
+++ b/meson.build
@@ -16,10 +16,9 @@ project(
     'build.rust_std=2021',
   ],
 )
-add_project_link_arguments(
-  '-llog',
-  language : ['c', 'cpp'],
-)
+if host_machine.system() == 'android'
+  add_project_link_arguments('-llog', language : ['c', 'cpp'])
+endif
 project_version = meson.project_version()
 
 if host_machine.system() == 'darwin'
